sdcm/provision/azure/utils.py

```diff
@@ -39,10 +39,10 @@ def get_scylla_images(  # pylint: disable=too-many-branches,too-many-locals
     if len(version_bucket) == 1:
         if '.' in scylla_version:
             # Plain version, like 4.5.0
-            tags_to_search['ScyllaVersion'] = lambda ver: ver and ver.startswith(scylla_version)
+            tags_to_search['scylla_version'] = lambda ver: ver and ver.replace("~", "-").startswith(scylla_version)
```

Comment:
what happens with an older image? it will still have camelCase, but the snake_case don't be found... shouldn't we support (for now) both cases? or be sure we have all older images updated with the "new" tag format?

Reply:
yes, we support both ways. See in line 70: 
`image.tags["scylla_version"] = image.tags.get('scylla_version', image.tags.get('ScyllaVersion'))` in case there's no `scylla_version` tag we use `ScyllaVersion`.